tests/Unit/API/Site/Controllers/Ads/RecommendationsControllerTest.php

```diff
@@ -119,23 +119,6 @@ public function test_get_recommendations_returns_empty_array_when_filter_by_non_
 		$this->assertEquals( 'rest_invalid_param', $response->get_data()['code'] );
 	}
 
-	public function test_get_recommendations_returns_error_if_account_not_connected() {
-		$this->account->method( 'get_connected_account' )
-			->willReturn( [ 'status' => 'not_connected' ] );
```

Comment:
@mukeshpanchal27 this is also used in other tests in the file, do we need to remove those as well?

Reply:
Removed in https://github.com/woocommerce/google-listings-and-ads/pull/3098/commits/f621188122168c9d0c779092378bf267c45ea625